routes/story.js

```diff
@@ -1,39 +1,61 @@
 const express = require("express");
 const router = express.Router();
+const Story = require('../models/Story');
 
 // @desc Story summary
 // @route GET /story/:storyId
 // @access Public
 router.get("/:storyId", (req, res) => {
-  res.send("fetch story details with that params.storyId");
+    const { storyId } = req.params.storyId;
+    try {
+        Story.findOne({ _id: storyId }, (err, story) => {
+            if (!story) throw "Invalid Story ID";
+        }).lean();
+        res.send("fetch story details with that params.storyId");
+    } catch (err) {
+        console.log(err);
+    }
 });
 
 // @desc Story Vote update
 // @route PUT /story/:storyId/vote
 // @access Private
 router.put("/:storyId/vote", (req, res) => {
-  res.send("increase the story vote counter");
+    const { storyId } = req.params.storyId;
+
+    try {
+        Story.findOne({ _id: storyId }, (err, story) => {
+            if (!story) throw "Invalid Story ID";
+        });
+
+        var voteCount = story.voteCount + 1;
+        var newvalues = { $set: { voteCount: voteCount } };
+        Story.updateOne({ ...req.body, storyId }, { _id: storyId }, newvalues);
+        res.send("increase the story vote counter");
```

Comment:
Again you are not sending the result in a response, it is just a string which I wrote by default

Reply:
Changes incorporated